.github/process_commit.py

```diff
@@ -36,15 +37,18 @@
     "perf",
     "other",
 }
+GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
```

Comment:
Is a maintainer expected to run this script manually and locally?
If so, I think it's better to 
- fail if the env var is not defined, and
- provide `--help` option and the `--help` option to work even when GITHUB_TOKEN is not set.

Reply:
this script shouldn't be run by maintainers, but by the github workflow when there is a push to main. the github token is set [here](https://github.com/pytorch/audio/blob/e8f7e192d4b47f0ec25b4920bd29ffe216739ebe/.github/workflows/pr-labels.yml#L26)